profile/README.md

```diff
@@ -79,10 +77,17 @@ Popular application forks with NeoFS support added:
 
 # NeoGo
 
-Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's
-all there. [ansible-neogo-n3](https://github.com/nspcc-dev/ansible-neogo-n3)
-can help you running it.
+Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's all there.
 
-To learn more about Neo smart contract development in go take a look at [neo-go-sc-wrkshp](https://github.com/nspcc-dev/neo-go-sc-wrkshp).
+To learn more about Neo smart contract development in go take a look at
+[neo-go-sc-wrkshp](https://github.com/nspcc-dev/neo-go-sc-wrkshp).
 
 There is also a Neo performance testing tool, [neo-bench](https://github.com/nspcc-dev/neo-bench).
+
+
+# Ansible
```

Comment:
I'm not sure we need a new top-level section for it. The idea behind this document is that we have two main things --- NeoFS (which has a lot of moving parts in it) and NeoGo. All of the associated things should be bound to one of these entities. Likely we can do the same here.

Reply:
This document was highly complicated. It's hard to read a lot of repeating words, like ansible ... ansible-role ... managed. So I simplified it, moving all about ansible into separate part.


Anything not clear what you suggest here.

Reply:
Moreover, I believe ansible part should be removed from this page, see #6